### PR TITLE
Improve BDN benchmarking guide

### PR DESCRIPTION
**Pull request by MarcoRossignoli**
Improve 'Benchmarking .NET Core applications' guide.
I followed 'Recommended workflow' section and added startup sample, to me is more clear and complete.
It's not immediatly clear understand how to start benchmarks that command line will filter.
Feel free to reject if you think it's unuseful.

PS. Works like a charm

/cc @adamsitnik

### CODE REVIEW

File: Documentation/project-docs/benchmarking.md [hunk 1]
```diff
@@ -52,15 +52,24 @@ If you want to run your benchmarks without spawning a new process per benchmark
...
 
         C:\Projects\corefx\bin\testhost\netcoreapp-Windows_NT-Release-x64\shared\Microsoft.NETCore.App\9.9.9\CoreRun.exe
 
 2. Create a new .NET Core console app using your favorite IDE
 3. Install BenchmarkDotNet (0.11.1+)
-4. Define the benchmarks
+4. Define the benchmarks and setup runner
+
+```cs
+...
```
Comment (adamsitnik) [inline Documentation/project-docs/benchmarking.md]: I think that the `...` could be removed so only "```" would stay

File: Documentation/project-docs/benchmarking.md [hunk 2]
```diff
@@ -52,15 +52,24 @@ If you want to run your benchmarks without spawning a new process per benchmark
...
+        C:\Projects\corefx>build.cmd -release -buildArch=x64
 
 After that, you should be able to find `CoreRun.exe` in a location similar to:
 
         C:\Projects\corefx\bin\testhost\netcoreapp-Windows_NT-Release-x64\shared\Microsoft.NETCore.App\9.9.9\CoreRun.exe
 
 2. Create a new .NET Core console app using your favorite IDE
 3. Install BenchmarkDotNet (0.11.1+)
-4. Define the benchmarks
+4. Define the benchmarks and setup runner
```
Comment (adamsitnik) [inline Documentation/project-docs/benchmarking.md]: maybe Define the benchmarks and **pass the arguments to BenchmarkSwitcher** would be better?

File: Documentation/project-docs/benchmarking.md (hunk 1 above)
Comment (MarcoRossignoli) [reply Documentation/project-docs/benchmarking.md]: done

File: Documentation/project-docs/benchmarking.md (hunk 2 above)
Comment (MarcoRossignoli) [reply Documentation/project-docs/benchmarking.md]: done

**Comment by MarcoRossignoli** [comment]
It's my pleasure thank's for this feature.

**Pull request merged**